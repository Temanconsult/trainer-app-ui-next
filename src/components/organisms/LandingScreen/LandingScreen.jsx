import React from 'react'
import LandingScreenHeader from '../../molecules/LandingScreenHeader'
import LandingScreenJumbotron from '../../atoms/LandingScreenJumbotron/LandingScreenJumbotron'

const LandingScreen = () => {
  return (
    <>
      <LandingScreenHeader />
      <LandingScreenJumbotron />
    </>
  )
}

export default LandingScreen
