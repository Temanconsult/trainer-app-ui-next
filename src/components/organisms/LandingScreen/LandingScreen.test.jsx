import React from 'react'
import { render } from '@testing-library/react'
import LandingScreen from '.'

describe('LandingScreen', () => {
  it('creates landing screen', () => {
    const { asFragment } = render(<LandingScreen />)
    expect(asFragment()).toMatchSnapshot()
  })
})
