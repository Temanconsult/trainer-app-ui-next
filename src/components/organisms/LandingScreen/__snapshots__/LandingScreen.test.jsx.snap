// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingScreen creates landing screen 1`] = `
<DocumentFragment>
  <nav
    class="mt-sm-4 navbar navbar-expand-lg navbar-light bg-light"
  >
    <span
      class="navbar-brand"
    >
      Best Tutors
    </span>
    <button
      aria-controls="basic-navbar-nav"
      aria-label="Toggle navigation"
      class="navbar-toggler collapsed"
      type="button"
    >
      <span
        class="navbar-toggler-icon"
      />
    </button>
    <div
      class="navbar-collapse collapse"
      id="basic-navbar-nav"
    >
      <div
        class="mr-auto navbar-nav"
      >
        <a
          class="nav-link"
          href="#"
          role="button"
        >
          Home
        </a>
        <a
          class="nav-link"
          href="#"
          role="button"
        >
          About
        </a>
        <a
          class="nav-link"
          href="#"
          role="button"
        >
          Courses
        </a>
        <a
          class="nav-link"
          href="#"
          role="button"
        >
          Tutors
        </a>
      </div>
      <div
        class="navbar-nav"
      >
        <a
          class="nav-link"
          href="#"
          role="button"
        >
          Become and Instructor
        </a>
        <button
          class="mr-lg-3 mb-lg-0 mb-sm-3 btn btn-outline-primary"
          type="button"
        >
          Login
        </button>
        <button
          class="btn btn-primary"
          type="button"
        >
          SignUp
        </button>
      </div>
    </div>
  </nav>
  <div
    class="jumbotron"
  >
    <h1>
      Learn with Ease and at your own pace
    </h1>
    <p>
      This is a simple hero unit, a simple jumbotron-style component for calling extra attention to featured content or information.
    </p>
    <p>
      <button
        class="btn btn-primary"
        type="button"
      >
        Learn more
      </button>
    </p>
  </div>
</DocumentFragment>
`;
